### PR TITLE
adding latest moderation BPs

### PR DESCRIPTION
**Pull request by parispittman**

### CODE REVIEW

File: communication/zoom-guidelines.md [hunk 1]
```diff
@@ -1,65 +1,87 @@
...
+* using a meeting that was set up through the "Meeting" tab in the zoom account and NOT the personal meeting ID
 
-You can also put an attendee on hold. This allows the host(s) to put attendee on hold to temporarily remove an attendee from the meeting. 
+After the meeting has started:
+* Assign a cohost to help with moderation. It should never be your notetaker unless it's a very small group.
+* Turn off screen sharing for everyone and indicate only host. If you have others that need to share their screen, the host can enable that on the fly. (via the ^ next to Share Screen)
+
+If you're dealing with a troll or bad actor:
+* You can put an attendee on hold. The participant will be put into a 'waiting room' and not have ability to chat or discuss written or verbally until the host undoes the hold.
+* Remove the participant. This is permanent. They will never be able to come back into that meeting ID on that device. Do not joke around with this feature; it's better to use the hold first and then remove.
```
Comment (jdumars) [inline communication/zoom-guidelines.md]: Maybe: Please be cautious when testing or using this feature, as it is permanent.

**Comment by jdumars** [comment]
One nit, otherwise /lgtm

**Comment by parispittman** [comment]
@jdumars addressed the nit - ready for your lgtm 😎

**Comment by jdumars** [comment]
/lgtm
thanks Paris for nailing this down!

**Pull request merged**